addons/website_sale/static/src/chatter/frontend/chatter_patch.js

```diff
@@ -0,0 +1,11 @@
+import { Chatter } from "@mail/chatter/web_portal/chatter";
+import { patch } from "@web/core/utils/patch";
+
+const chatterPatch = {
+    get requestList() {
+        return this.props.threadModel === "product.template"
+            ? [...super.requestList, "ratingStats"]
+            : super.requestList;
+    },
```

Comment:
Shouldn't it only be done if the requestList is coming from a portal chatter thread?

Similar comment as before, where condition should be based on code rather than relying on bundle loading files or not.